### PR TITLE
Fix/dropdown item size

### LINKED ISSUE #1039: [BUG]Dropdown Select not working in Firefox browser
**Describe the bug**
Using the Firefox browser, when selecting an item, the item is not selected.

**To Reproduce**

1. Go to https://cookbook.kirby.design/home/showcase/dropdown
2. Click on 'Dropdown with plain text"
3. Select 'Item 2'
4. The item is not selected

**Kirby version**
- 1.2.25

**Expected behavior**
The item should be selected

**Screenshots**

**Please complete the following information:**
- OS: Windows
- Browser: Firefox
- Browser version [e.g. 22]

**Additional context**

## Tasks:
- [ ] Move issue from "Ready to do" -> "in progress" on our [project board](https://github.com/kirbydesign/designsystem/projects/1)
- [ ] Create Fix Branch
- [ ] Create test reproducing the error
- [ ] Fix
- [ ] If relevant [Update Examples and Showcase](https://cookbook.kirby.design/home/showcase/button)
- [ ] Push commits to GitHub
- [ ] Open a pull request (remember to add `closes #issueno` to the description) and ask @kirbydesign/kirby-guild for review
- [ ] Merge to master and delete branch

### LINKED ISSUE #1047: [BUG] - Dropdown not saving selected item
**Describe the bug**
Occasionally the selected item is not reflected in the button text after the dropdown is collapsed - the previous item is still shown.
 
**To Reproduce**

1. Go to kirby cookbook dropdown example - https://cookbook.kirby.design/home/showcase/dropdown
2. Click on dropdown button
3. Choose item

**Kirby version**
- 1.2.31

**Expected behavior**
It should always remember my selection

**Screenshots**

**Please complete the following information:**
- OS: MacOS (browser) and iOS (native)
- Browser Chrome
- Browser version 81

**Additional context**

Important as it affect UX in production

## Tasks:
- [ ] Move issue from "Ready to do" -> "in progress" on our [project board](https://github.com/kirbydesign/designsystem/projects/1)
- [ ] Create Fix Branch
- [ ] Create test reproducing the error
- [ ] Fix
- [ ] If relevant [Update Examples and Showcase](https://cookbook.kirby.design/home/showcase/button)
- [ ] Push commits to GitHub
- [ ] Open a pull request (remember to add `closes #issueno` to the description) and ask @kirbydesign/kirby-guild for review
- [ ] Merge to master and delete branch

### PR DESCRIPTION
**Pull request by jakobe**
This PR closes #1039  and closes #1047 
Also sets item size inside dropdown to `sm` (44px) and allows for 8 items before scrollable dropdown.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts [hunk 1]
```diff
@@ -985,4 +984,66 @@ describe('DropdownComponent', () => {
...
+         </kirby-dropdown>`,
+        {
+          props: {
+            items: items,
+          },
+        }
+      );
+    });
+
+    it('should have correct item size xxxx', () => {
```
Comment (askarby) [inline libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts:1019]: Is the `xxxx` on purpose?

File: libs/designsystem/src/lib/scss/base/_variables.scss [hunk 2]
```diff
@@ -98,3 +98,17 @@ $elevations: (
 ****************************************************************************/
 $alert-max-width: 359px;
 $compact-modal-max-width: 359px;
+
+/* Item
+****************************************************************************/
+$item-sizes: (
```
Comment (askarby) [inline libs/designsystem/src/lib/scss/base/_variables.scss:104]: Of very little importance....but.... it's not clear if it's height?.... size is a somewhat vague name?

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts (hunk 1 above)
Comment (jakobe) [reply libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts:1022]: My bad 🤦 
Removed!

File: libs/designsystem/src/lib/scss/base/_variables.scss (hunk 2 above)
Comment (jakobe) [reply libs/designsystem/src/lib/scss/base/_variables.scss]: Renamed 👌

**Pull request merged**